_posts/2022-11-10-why-python-package-health-metrics-matter.md

```diff
@@ -0,0 +1,111 @@
+
+If you have evaluation or education in your professional background like I do, 
+you may even
+[create a logic model](https://thecompassforsbc.org/how-to-guide/how-develop-logic-model-0) to map activities to outcomes and goals. This logic 
+model helps you define how 
+to collect the data that you need to track outcome success. 
+
+## Baseline data are critical to collect at the start of building a program or organization
+
+As I am building the pyOpenSci program, I find myself thinking about what metrics 
+around Python scientific open source software we want to track to better understand:
+
+1. The outcomes of our activities
+2. The overall health of packages in our growing pyOpenSci ecosystem (specific to our organization)
+3. How/if we contributed to improving the health of packages in our ecosystem
+4. How we are impacting the broader scientific python, open source community
+
+### Baseline metrics collected from the start will help your future self when running or working in an organization
+
+As mentioned above, collecting metrics from the start 
+of your efforts allows you to get off the ground running with data that you can use to compare to future data. Thus while it may not be the work that you want to do, it will 
+help your future self. 
+
+For pyOpenSci, collecting metrics allows us in the future 
+to evaluate our programs and adaptively change things to 
+make sure we are getting the outcomes that we want. 
+
+Outcomes such as 
+* Scientists being better able to find packages that are maintained to support their workflows  
+* Improved documentation in packages as a result of our reviews
+
+### But what metrics should we collect about scientific Python packages?
+
+[In a previous post,](/blog/2022-10-24-why-should-python-open-source-matter-science) I spoke 
+generally about why open source should matter to you as a scientist and as a 
+developer or package maintainer. 
+
+To better understand what data we should be collecting to track our packages 
+health over time, I went to Twitter to see what my colleagues around the world
+had to say. That conversation resulted in some really interesting insights.
+
+Most importantly, it allowed me to begin to break down and group metrics in 
+terms of pyOpenSci goals. 
+
+
+### Goals for package metrics
+
+We hope that:
+
+* Peer review improves Python package structure and usability. 
+* Peer review in some way supports maintenance and/or responsible archiving when a package comes to life-end.
+* Over time, the package is improved and maintained with possible contributions for those other than the maintainer. 
+
+We need metrics to understand things like
+
+* Community adoption of the package (are scientists using it?)
+* Maintenance level of the package (are maintainers still working on it and fixing bugs?)
+* Infrastructure (are tests setup to help identify if contributions break things? )
```

Comment:
```suggestion
* Infrastructure (are tests set up to help identify if contributions break things? )
```
Still reading overall (sounds great far!) but just a very minor suggested edit.

I am pretty sure that when using "set up" as a phrasal verb and not a noun, you want a space between the two words. 

It's confusing here because of the copula "are", that "set up" is modifying as a helper, kind of like a gerund, "are test running? tests are running". 

You would write "Tests are set up to", not "test are setup to"; you wouldn't write "I am setup to fail"

I don't find any good prescriptive grammarian blog posts to tell me if I'm right, but

"are set up to" returns [43.8 M results](https://www.google.com/search?q=%22are+set+up+to%22&sxsrf=ALiCzsbLRsvFrbJkVkkTWXzPpg_wKFLv6Q%3A1667495747932&source=hp&ei=Q_djY4LYNdTZ9API-7OoCQ&iflsig=AJiK0e8AAAAAY2QFUwrbMO9LLNMI0lO9J2pmgLgQFWg1&ved=0ahUKEwjChd7dwZL7AhXULH0KHcj9DJUQ4dUDCAk&uact=5&oq=%22are+set+up+to%22&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEBYQHjIGCAAQFhAeMggIABAWEB4QDzIGCAAQFhAeMgUIABCGAzIFCAAQhgM6BAgjECc6BQgAEJECOggILhDUAhCRAjoRCC4QgwEQxwEQsQMQ0QMQgAQ6CwgAEIAEELEDEIMBOg4ILhCABBCxAxDHARDRAzoECAAQQzoICAAQgAQQsQM6CggAELEDEIMBEEM6BQgAEIAEOgcIABCxAxBDOhEILhCABBCxAxCDARDHARDRAzoICC4QsQMQgwE6CwguEIAEEMcBEK8BOgsILhCABBCxAxCDAToICC4QgAQQsQM6CAgAELEDEIMBOgsILhCABBCxAxDUAjoICAAQsQMQkQI6BQguEIAEOggIABAWEB4QClAAWI0ZYPgaaAZwAHgAgAG8AYgB1hSSAQQxLjIwmAEAoAEB&sclient=gws-wiz)
"are setup to" return [195k results](https://www.google.com/search?q=%22are+setup+to%22&sxsrf=ALiCzsblv7LzR7ffOBefwkolNZHe20I1mg%3A1667495752615&ei=SPdjY5eLJfCHptQP0bSNuAM&ved=0ahUKEwiXz_7fwZL7AhXwg4kEHVFaAzcQ4dUDCA8&uact=5&oq=%22are+setup+to%22&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIGCAAQBxAeMgoIABAIEAcQHhAPMgUIABCGAzIFCAAQhgMyBQgAEIYDOgoIABBHENYEELADSgQIQRgASgQIRhgAUPgEWPgEYO0GaAFwAXgAgAGcAYgBnAGSAQMwLjGYAQCgAQHIAQXAAQE&sclient=gws-wiz-serp)